include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (404)
+#define OPENTHREAD_API_VERSION (405)
```

Comment:
Can you rebase an increment this number again? Thanks!

Reply:
Sure.

Reply:
Done. Do we need to re-run the whole validation again?